README.md

```diff
@@ -68,7 +69,8 @@ The Access Key is contained in credentials.csv.
 
 **Note well:** credentials.csv contains `secrets`!
 Do not share them or store them in git.
-The [.gitignore](.gitignore) in this repository explicitly ignores this file.
+The [.gitignore](.gitignore) in this repository explicitly ignores this file. Altering the [.gitignore](.gitignore) file
+in this repository could result in your AWS credentials (credentials linked to *your* credit card information) being visible on Github. *NEVER COMMIT SECRETS TO GIT*
 
```

Comment:
Just saw the merge :( You were right to merge, I should have followed my instinct and not accommodated the issue in the first place, but I'd like to talk about this handholding balance at some point @gaand maybe with @jrhorn424 and @payne-chris-r as well. 

I can understand wanting emphasis here because ignoring instructions can result in actual financial loss, but all instructions should be followed with that level of scrutiny and sense of importance.  I know there's a time and place for hand-holding, but I don't think this was it.